Update docstring and readme (#9)
ynput/ayon-speedtree: README.md

# SpeedTree addon

This adds Unity SpeedTree integration for AYON. SpeedTree is the industry-standard vegetation toolkit for projects of any size and style, used across both games and film.

## Dev resources

* https://docs.unity3d.com/speedtree-runtime-sdk/manual/index.html

> * https://docs.unity3d.com/speedtree-pipeline-sdk/manual/index.html

Reviewer: This link is broken. It's also broken on their website. 
<img width="620" height="141" alt="image" src="https://github.com/user-attachments/assets/89ae05d9-8664-4812-bb32-79d7459b3a50" />

## Settings

> Path to Zbrush executable must be set in the Ayon Setting in `Applications` addon (`ayon+settings://applications/applications/speedtree`) and added in `Anatomy`.`Attributes` for particular project to be visible in the Launcher.

Reviewer: ```suggestion
Path to SpeedTree executable must be set in the Ayon Setting in `Applications` addon (`ayon+settings://applications/applications/speedtree`), for more information see [Applications and Tools](https://help.ayon.app/articles/3945756-applications-and-tools) documentation.
```

### Implemented workflows

> Currently supports importing/exporting models and saving/opening/publishing workfiles in SpeedTree integration. All the associated data would be stored in `.sptree_metadata` folder

Reviewer: This is a question more than a review so that I can update the docs accordingly...
Does this work similar to zbrush?  
https://help.ayon.app/articles/2471745-working-with-zbrush-in-ayon#0ki4x8mb45l

Author: In terms of having metadata folder to store all the ayon-related data, yes

> ## How to start

Reviewer: ```suggestion
## How Create Addon Zip Package
```

There is a `create_package.py` python file which contains logic how to create the addon from AYON codebase. Just run the code.
```shell

> python ./create_package.py

Reviewer: this one is miss closing three backticks `

> 

Reviewer: ```suggestion

## Additional Resources
For information about working with speedtree in AYON, see
- Artist Documentation: [Working with SpeedTree in AYON](https://help.ayon.app/en/help/articles/1430359-working-with-speedtree-in-ayon)
- Admin Documentation: [Configure SpeedTree Addon](https://help.ayon.app/en/help/articles/9963950-configure-speedtree-addon)

```
